config.c

```diff
@@ -967,13 +966,16 @@ static int readConfigFile(const char *configFile, struct logInfo *defConfig)
         return 0;
     }
 
-    if (!(pw = getpwuid(getuid()))) {
-        message(MESS_ERROR, "Logrotate UID is not in passwd file.\n");
+    if (!getpwuid(getuid())) {
+        message(MESS_ERROR, "Can not find logrotate UID (%d) in passwd file: %s\n",
```

Comment:
Although I am not a native speaker, I believe that "can not" should rather be spelled as "cannot": https://www.grammarly.com/blog/cannot-or-can-not/

I see we have the same pattern in all the "can not fork" messages, which I did not spot earlier.